src/screens/login/components/Login/index.js

```diff
@@ -43,9 +44,14 @@ module.exports = class Login extends StrangeForms(React.Component) {
         });
     }
 
+    componentDidMount() {
+
+        this.props.clearErrors();
+    }
+
     inputsAreValid() {
 
-        return this.state.email !== '' && this.state.password !== '';
+        return this.state.email !== '' && this.state.password !== '' && !this.showEmailError();
```

Comment:
This pretty much addresses my comment from an earlier PR.  I don't know if the email field being blurred needs to be taken into account here, but I also don't see it changing the behavior here.